### PR TITLE
Feature/table view

### PR DESCRIPTION
**Pull request by wwwsurenio**
I added the TableView to the app, and set it up to show the movie titles for now, next step is to setup the custom cell and show all the data from the array.
Or is there another way I should be doing this?
Simulator Screen Shot - iPhone 14 - 2023-05-12 at 14 12 30

### CODE REVIEW

File: imdb/UI/View/MovieViewController.swift [hunk 1]
```diff
@@ -27,73 +22,61 @@ class MovieViewController: UIViewController {
...
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        // Здесь настройка ячейки
+        let cell = tableView.dequeueReusableCell(withIdentifier: "cellId", for: indexPath)
+        let currentLastItem = movies[indexPath.row]
+        cell.textLabel?.text = currentLastItem.movieTitle
+        return cell
+    }
+    
+    func createMovieArray() {
+        movies.append(MovieModel(
```
Comment (aleksandr-lavrinenko) [inline imdb/UI/View/MovieViewController.swift]: Можем пока создать переменную let movies: [MovieModel] = [
MovieModel(...),
MovieModel(..)
]

File: imdb/UI/View/MovieViewController.swift [hunk 2]
```diff
@@ -27,73 +22,61 @@ class MovieViewController: UIViewController {
...
-            ratingContainerView.widthAnchor.constraint(equalTo: ratingLabel.widthAnchor),
-            ratingLabel.widthAnchor.constraint(equalToConstant: 30)
+            tableView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor),
+            tableView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor),
+            tableView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 8),
+            tableView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor)
         ])
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "cellId")
```
Comment (aleksandr-lavrinenko) [inline imdb/UI/View/MovieViewController.swift]: Все правильно - теперь попробуй UITableViewCell заменить на MovieCell!

**Comment by wwwsurenio** [comment]
**Created a custoom cell - MovieTableCell**
Added properties for each movie in the table view cell.
Added private variables to organize and configure each element within the table view cell.
Created and used the anchor function to set up constraints for the layout of the table view cell's content.
Removed unnecessary comments.

Code refactored based on comments above.

Current results:
Simulator Screen Shot - iPhone 14 - 2023-05-13 at 21 50 32

**Comment by wwwsurenio** [comment]
Added more films to test scrolling
Simulator Screen Shot - iPhone 14 - 2023-05-13 at 22 33 14

**Review by aleksandr-lavrinenko — APPROVED** [review]

**Pull request merged**